src/cov.jl

```diff
@@ -44,80 +37,73 @@ that the data are centered and hence there's no need to subtract the mean.
 When `vardim = 1`, the variables are considered columns with observations in rows;
 when `vardim = 2`, variables are in rows with observations in columns.
 """
-function scattermat end
```

Comment:
Right, but is that line even necessary (calling help on the method name still works the same). Also, the other line is just `cov` vs `function cov end`, what would the preference be to maintain consistency?